nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ServerSynchronizer.java

```diff
@@ -317,7 +316,7 @@ private boolean syncFile(URI root, File base, ModTime modTime, Payload.Outbound
             if (logger.isLoggable(FINE))
                 logger.log(FINE, "ServerSynchronizer: sending file {0}{1}",
                         new Object[] { f, modTime.time == 0 ? " because it doesn't exist on the instance" : " because it was out of date" });
-            payload.requestFileReplacement(MediaType.OCTET_STREAM.toString(), root.relativize(f.toURI()), "configChange", null, f, true);
+            payload.requestFileReplacement("application/octet-stream", root.relativize(f.toURI()), "configChange", null, f, true);
```

Comment:
There is `javax.ws.rs.core.MediaType.APPLICATION_OCTET_STREAM` if that is accessible in this context.